extensions/hibernate-orm/deployment/src/main/java/io/quarkus/hibernate/orm/deployment/HibernateOrmProcessor.java

```diff
@@ -104,8 +105,9 @@
     HibernateOrmConfig hibernateConfig;
 
     @BuildStep
-    HotDeploymentWatchedFileBuildItem configFile() {
-        return new HotDeploymentWatchedFileBuildItem("META-INF/persistence.xml");
+    List<HotDeploymentWatchedFileBuildItem> hotDeploymentFiles() {
+        return Arrays.asList(new HotDeploymentWatchedFileBuildItem("META-INF/persistence.xml"),
+                new HotDeploymentWatchedFileBuildItem(getSqlLoadScript()));
```

Comment:
This means will we also produce a `HotDeploymentWatchedFileBuildItem` if `no-file` is provided as  sql load script?

Reply:
Yes and that seems pretty pointless, I'm updating the PR again, thanks!